### PR TITLE
Proxyingandhooking

### PR DESCRIPTION
**Pull request by realswat1**
This PR is to allow proxying, and webhooking. This needed to ensure that the application constantly gets up to date info from the mux api. After this we plan to add the mux player to the main page . 

I used resources from hoodeck for the proxy (hookdeck CLI), mux api.

Below shows first hitting the /broadcast endpoint to create the video data which would have is_live as false by default the starting a screen using the stream_key in OBS and if we have a working hook, the is _live changed to true.
https://www.loom.com/share/24d501399b004fb6871730b8e7c34133?sid=00902010-8fc3-4943-b44c-28006bbe8121

### CODE REVIEW

File: exposee-app/server.js [hunk 1]
```diff
@@ -173,34 +174,35 @@ app.post('/videos', validate_Token,async (req, res) => {
 app.post('/broadcast',validate_Token, async (req, res)=> {
   try {
     const { description } = req.body;
-
-    const user_Id = req.userId; 
+    const user_Id = req.userId; // Use req.userId here instead of req.userId
```
Comment (alfonsocheng) [inline exposee-app/server.js]: what does this comment mean?

File: exposee-app/server.js [hunk 2]
```diff
@@ -19,12 +19,42 @@ import dotenv from 'dotenv';
...
+      api_key: API_KEY_FROM_REQUEST, 
+      is_live: true,
+      is_saved: true,
+      mux_stream_key: streamKey, 
+      mux_playback_id: playbackId, 
+    });
+
+    return liveStream;
+  } catch (error) {
+    console.error('Error saving stream details to the database:', error);
```
Comment (enniefu) [inline exposee-app/server.js]: do we want to show alert here to notify user the failure?

File: exposee-app/server.js [hunk 3]
```diff
@@ -19,12 +19,42 @@ import dotenv from 'dotenv';
...
+    return null;
```
Comment (enniefu) [inline exposee-app/server.js]: what will happen if we return null here? do you handle it in the downstream?

File: exposee-app/server.js (hunk 2 above)
Comment (enniefu) [inline exposee-app/server.js]: do we want to show alert here to notify user the failure?

**Pull request merged**